tools/conformance.py

```diff
@@ -636,7 +650,7 @@ class Webdriver(object):
         'chrome_public': 'org.chromium.chrome',
     }
 
-    def __init__(self, path, browser, host_os, target_os, android_device=None, debug=False):
+    def __init__(self, path, browser, host_os, target_os, android_device=None, debug=False, tools=False):
         self._logger = Util.get_logger()
         self.path = path
         self.target_os = target_os
```

Comment:
I couldn't add comment in below code, but Safari webdriver path can be initialized the similar way with ChromeOS as below. Safari is a bit complicated as you have 2 webdrivers.